editor/store/reducer.js

```diff
@@ -696,14 +687,13 @@ export function metaBoxes( state = defaultMetaBoxState, action ) {
 }
 
 // Create responsive reducer with the breakpoints imported from the scss variables file.
```

Comment:
I think this comment is now incorrect given that you remove the breakpoints from the code.